### ISSUE TITLE
Clarification on : in See Also.

### ISSUE THREAD

**Issue by Carreau** [numpy/numpydoc#344]
```
If the combination of the function name and the description creates a line that is too long, the entry may be written as two lines, with the function name and colon on the first line, and the description on the next line, indented four spaces:
```

How strict should we be with the colon ? It seem like numpy almost never put it in multiline, 
and this is a bit incoherent with parameter section, where the colon is necessary if and only if there is a type parameter.

Should the advice to put a colon be removed ? 
Or is it better to alway pt a colon to make the parsing easier ?

**Comment by rgommers** [comment]
> If the combination ....

`You made that line too long on purpose didn't you` :)

I'm not sure, it's just advice - I'd say copy a few representative examples that we consider good style, rather than make it clear by a description like this.

**Comment by Carreau** [comment]
Sorry, when typing it was wrapped by the edit box, fixed. and s/numpy/scipy/g

```
@contextlib.contextmanager
def reset_state():
    """
    Returns a context manager that resets all state once exited.

    See Also
    --------
    set_state
        Context manager that sets the backend state.
    get_state
        Gets a state to be set by this context manager.
"""
```

And my main question is to know wether I should add an option to my docstring reformatter  to not put colon when reformatting on two lines.

Visually it is not bothering me, it just that once parsed by numpydoc the info of wether it was multiline or nor, or wether it there was a colon or not.

So while i'm digging into numpydoc, should I maybe emit a warning if `:` is missing ? 

It is also a harder to rewrite a different parser for numpydoc when the "spec" is loose.

**Comment by rgommers** [comment]
> And my main question is to know wether I should add an option to my docstring reformatter to not put colon when reformatting on two lines.

I agree keeping the `:` is more consistent - if your tooling can fix up the formatting in NumPy, SciPy etc. easily, then I'm fine with making that choice and emitting a warning.
